File: Beecrowd/Beginner/1000 - 1099/1043 - Triangle/Python/1043_Triangle.py
a, b, c = map(float, input().split())

if a + b > c and a + c > b and b + c > a:
    print(f"Perimetro = {a + b + c:.1f}")
else:
    print(f"Area = {(a + b) * c / 2.0:.1f}")

File: Beecrowd/1 - Beginner/1000 - 1099/1043 - Triangle/1043_Triangle.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    double a, b, c;
    cin >> a >> b >> c;

    if (a + b > c && a + c > b && b + c > a) {
        cout << "Perimetro = " << fixed << setprecision(1) << a + b + c << endl;
    } else cout << "Area = " << fixed << setprecision(1) << (a + b) * c / 2.0 << endl;
    
    return 0;
}